projects/storefrontlib/src/cms-structure/seo/structured-data/builders/product/jsonld-product-review.builder.ts

```diff
@@ -12,21 +13,25 @@ import { JsonLdBuilder } from '../schema.interface';
   providedIn: 'root',
 })
 export class JsonLdProductReviewBuilder implements JsonLdBuilder<Product> {
-  constructor(private reviewService: ProductReviewService) {}
+  constructor(
+    protected reviewService: ProductReviewService,
+    protected config: SeoConfig
+  ) {}
 
   build(product: Product): Observable<any> {
     return this.reviewService.getByProductCode(product.code).pipe(
-      filter(Boolean),
       map((reviews: Review[]) => {
-        return {
-          aggregateRating: this.buildAggregatedReviews(product, reviews),
-          review: reviews.map((review) => this.buildReviews(review)),
-        };
+        return reviews?.length > 0
```

Comment:
if we use conditional operator here, we can go with:
```
map(..) => reviews?.length > 0 ? .. : ..
```
(drop the `{}` and `return`)